### ISSUE TITLE
Webpack: Module parse failed using plugin

### ISSUE THREAD

**Issue by Nosfistis** [nestjs/swagger#476]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Using the plugin with Webpack throws me the following error:
```
ERROR in ./libs/entities/src/resource/resource.entity.ts 27:16
Module parse failed: Unexpected token (27:16)
File was processed with these loaders:
 * ./node_modules/ts-loader/index.js
You may need an additional loader to handle the result of these loaders.
|     static _OPENAPI_METADATA_FACTORY() {
```
This happens even if I add `@ApiProperty()` decorators manually everywhere.

Without the plugin build works ok! I have cross-checked for unresolved cyclical dependencies in the output schema, and it seems fine.

## Expected behavior

Using Webpack and libraries should work with @nestjs/swagger@4 library.

## Minimal reproduction of the problem with instructions

1. Create a new project.
1. Create a project library using the NestJS CLI.
1. Define some decorated entities in the library, and export them via an index file.
1. Try to build with swagger enabled.

## Environment

<pre><code>
Nest version: 6.10.14

 
For Tooling issues:
- Node version: 13.1.0  
- Platform: Windows

**Comment by kamilmysliwiec** [comment]
I can't reproduce your issue.

Please, provide a minimal repository which reproduces your issue.

**Comment by Nosfistis** [comment]
Hi, you are very much right and I am sorry for being vague. At that point, it was getting frustrating not to be able to identify what was wrong and create a proper reproduction without effectively duplicating my project.

I have found the root cause. It turns out, I was using a `private static` property in the class, having a util object there for typeorm's `ValueTransformer`. I guess that the plugin tries to reflect on that property, which is not a simple key (not actual part of the model), has a complex default value and is part of the static definition of the class.

The example entity is below. The only way of fixing this is removing the static property altogether outside the class, or placing its implementation directly on the `@Column` property.

```Typescript
@Entity()
export class Resource extends BaseEntity {

  private static decimalTransformer: ValueTransformer = {
    from: (value: number): number => Number(value),
    to: (value: any): any => value,
  };

  @ApiProperty({ maxLength: 50 })
  @Column({ type: 'varchar', length: 50 })
  title: string;

  @ApiProperty({ maxLength: 50 })
  @Column({ type: 'varchar', length: 50 })
  description: string;

  @ApiProperty({ enum: DataType, default: DataType.value })
  @Column({ type: 'enum', enum: DataType, default: DataType.value })
  dataType: DataType;

  @ApiProperty()
  @Column({ width: 1 })
  enabled: boolean;

  @ApiPropertyOptional({ readOnly: true })
  @Column({
    type: 'decimal',
    default: 0,
    precision: 15,
    scale: 5,
    readonly: true,
    nullable: true,
    transformer: Resource.decimalTransformer,
  })
  total?: number;

}
```

**Comment by kamilmysliwiec** [comment]
Fixed in 4.1.8. Static properties will be ignored now. Thanks for reporting
